docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM amd64/ubuntu:22.04
+
+RUN apt-get update && apt-get upgrade -y && apt-get upgrade -y binutils && (1 | apt-get upgrade -y tzdata) && apt-get install -y wget && wget https://s3.us-west-2.amazonaws.com/downloads.yugabyte.com/repos/reporpms/yb-apt-repo_1.0.0_all.deb && apt-get install -y ./yb-apt-repo_1.0.0_all.deb && apt-get clean && apt-get update && apt-get install -y yb-voyager && rm -rf /yb-apt-repo_1.0.0_all.deb && apt-get remove -y wget && apt-get clean -y
```

Comment:
Removed the upgrade command

`apt` tries to upgrade` tzdata `while installing `yb-voyager`. At that time it asks for prompts to select the timezone. Due to this the build fails. That's why I explicitly upgrade` tzdata `and feed it 1 to select the timezone.

apt-get update needs to be the first step as well. Otherwise it wont be able to find the wget package.